packages/grid/src/vaadin-grid-data-provider-mixin.js

```diff
@@ -96,6 +96,21 @@ export const ItemCache = class ItemCache {
     }
     return { cache: this, scaledIndex: thisLevelIndex };
   }
+
+  /**
+   * Gets the scaled index as flattened index on this cache level.
+   * In practice, this means that the effective size of any expanded
+   * subcaches preceding the index are added to the value.
+   * @param {number} scaledIndex
+   * @returns {number} The flat index on this cache level.
```

Comment:
nit: we use `@return` instead of `@returns` annotation.

```suggestion
   * @return {number} The flat index on this cache level.
```

Reply:
Done